{"sections":[],"kind":"symbol","identifier":{"url":"doc:\/\/me.digi.sdk.example.librarycore.DigiMeCore\/documentation\/DigiMeCore\/SDKError\/other","interfaceLanguage":"swift"},"abstract":[{"type":"text","text":"An unexpected error has occurred - please contact support"}],"variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/digimecore\/sdkerror\/other"]}],"hierarchy":{"paths":[["doc:\/\/me.digi.sdk.example.librarycore.DigiMeCore\/documentation\/DigiMeCore","doc:\/\/me.digi.sdk.example.librarycore.DigiMeCore\/documentation\/DigiMeCore\/SDKError"]]},"metadata":{"fragments":[{"text":"case","kind":"keyword"},{"kind":"text","text":" "},{"kind":"identifier","text":"other"}],"externalID":"s:10DigiMeCore8SDKErrorO5otheryA2CmF","role":"symbol","symbolKind":"case","modules":[{"name":"DigiMeCore"}],"title":"SDKError.other","roleHeading":"Case"},"schemaVersion":{"minor":3,"patch":0,"major":0},"primaryContentSections":[{"kind":"declarations","declarations":[{"platforms":["iOS"],"languages":["swift"],"tokens":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"text":"other","kind":"identifier"}]}]}],"references":{"doc://me.digi.sdk.example.librarycore.DigiMeCore/documentation/DigiMeCore/SDKError":{"kind":"symbol","fragments":[{"text":"enum","kind":"keyword"},{"text":" ","kind":"text"},{"text":"SDKError","kind":"identifier"}],"abstract":[{"text":"SDK Errors","type":"text"}],"type":"topic","role":"symbol","navigatorTitle":[{"text":"SDKError","kind":"identifier"}],"url":"\/documentation\/digimecore\/sdkerror","title":"SDKError","identifier":"doc:\/\/me.digi.sdk.example.librarycore.DigiMeCore\/documentation\/DigiMeCore\/SDKError"},"doc://me.digi.sdk.example.librarycore.DigiMeCore/documentation/DigiMeCore":{"title":"DigiMeCore","url":"\/documentation\/digimecore","type":"topic","kind":"symbol","role":"collection","identifier":"doc:\/\/me.digi.sdk.example.librarycore.DigiMeCore\/documentation\/DigiMeCore","abstract":[]},"doc://me.digi.sdk.example.librarycore.DigiMeCore/documentation/DigiMeCore/SDKError/other":{"type":"topic","abstract":[{"text":"An unexpected error has occurred - please contact support","type":"text"}],"url":"\/documentation\/digimecore\/sdkerror\/other","fragments":[{"kind":"keyword","text":"case"},{"text":" ","kind":"text"},{"kind":"identifier","text":"other"}],"identifier":"doc:\/\/me.digi.sdk.example.librarycore.DigiMeCore\/documentation\/DigiMeCore\/SDKError\/other","role":"symbol","kind":"symbol","title":"SDKError.other"}}}